docs/reference/ingest/processors/enrich.asciidoc

```diff
@@ -16,7 +16,20 @@ check out the <<ingest-enriching-data,tutorial>> to get familiar with enrich pol
 | `field`            | yes       | -                    | The field in the input document that matches the policies match_field used to retrieve the enrichment data.
 | `target_field`     | yes       | -                    | The field that will be used for the enrichment data.
 | `ignore_missing`   | no        | false                | If `true` and `field` does not exist, the processor quietly exits without modifying the document
-| `override`         | no        | true                 | If processor will update fields with pre-existing non-null-valued field. When set to `false`, such fields will not be touched.
-| `max_matches`      | no        | 1                    | The maximum number of matched documents to include under the configured target field. In order to avoid documents getting too large, the maximum allowed value is 128.
+| `override`         | no        | true                 | If processor will update fields with pre-existing non-null-valued field. When set to `false`, such fields will                                                             not be touched.
+| `max_matches`      | no        | 1                    | The maximum number of matched documents to include under the configured target field. In order to avoid                                                                    documents getting too large, the maximum allowed value is 128.
+| `shape_relation`   | no        | `INTERSECTS`         a| Spatial relation operator
+used to match the <<geo-shape,geoshape>> of incoming documents
+to documents in the enrich index.
+Valid operators are:
```

Comment:
do you think it would be relevant to point the user to the other relations that the geo_shape query supports?

The relation support is a little confusing due to limited support in the default indexing strategy, but that is probably even more reason to point to those docs

Reply:
Very good point. I've changed the description with 7ba28d77cf2bc0b37952c0ff3196b18064af8dc8:
https://github.com/elastic/elasticsearch/pull/47745/commits/7ba28d77cf2bc0b37952c0ff3196b18064af8dc8#diff-2b3db85450cced18529eb549903387f2R27

Let me know if you feel any other changes are needed.